packages/kbn-management/settings/components/field_row/__stories__/common.tsx

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import type { ComponentMeta } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import { EuiPanel } from '@elastic/eui';
+import { SettingType } from '@kbn/management-settings-types';
+
+import { KnownTypeToMetadata, UiSettingMetadata } from '@kbn/management-settings-types/metadata';
+import {
+  useFieldDefinition,
+  getDefaultValue,
+  getUserValue,
+} from '@kbn/management-settings-field-definition/storybook';
+import { FieldRow as Component, FieldRow } from '../field_row';
+import { FieldRowProvider } from '../services';
+import { OnChangeFn } from '../types';
+
+/**
+ * Props for a {@link FieldInput} Storybook story.
+ */
+export interface StoryProps<T extends SettingType>
+  extends Pick<KnownTypeToMetadata<T>, 'userValue' | 'value'> {
+  /** Simulate if the UiSetting is custom. */
```

Comment:
Is the only thing that this is doing is setting an icon w/ tooltip? Maybe worth adding more details about its behavior.

Reply:
This was updated with the latest commit.